awx/main/scheduler/task_manager.py

```diff
@@ -576,8 +576,9 @@ def schedule(self):
         with advisory_lock('task_manager_lock', wait=False) as acquired:
             with transaction.atomic():
                 if acquired is False:
-                    logger.debug("Not running scheduler, another task holds lock")
+                    logger.warning("Not running scheduler, another task holds lock")
```

Comment:
I imagine this happens a _lot_, and we expect that. I'd keep this lower than warning.

Reply:
Yep 👍

Reply:
sigh, ok, changed back.